browsers/node16.5.0-chrome94-ff93/README.md

```diff
@@ -17,4 +17,4 @@ A complete image with all operating system dependencies for Cypress, Chrome
 ```
 
 **Note:** this image uses the `root` user. You might want to switch to non-root
-user like `node` when running this container for security.
+user like `node` when running this container for security.
```

Comment:
Not sure why this is showing as changed as both of these lines look the same to me